superbench/benchmarks/micro_benchmarks/micro_base.py

```diff
@@ -70,21 +71,139 @@ def _process_numeric_result(self, metric, result):
         self._result.add_result(metric, sum(result) / len(result))
         return True
 
-    def _process_raw_result(self, raw_output):
+    def print_env_info(self):
+        """Print environments or dependencies information."""
+        # TODO: will implement it when add real benchmarks in the future.
+        pass
+
+
+class MicroBenchmarkWithInvoke(MicroBenchmark):
```

Comment:
Shall we add UT for this class in this PR?

Reply:
Done.